RazorGenerator.Core.v1/CodeTransformers/GenericParametersTransformer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RazorGenerator.Core
+{
+    public class GenericParametersTransformer : RazorCodeTransformerBase
+    {
+        readonly string[] _parameters;
```

Comment:
Old habits die hard :) Though I don't see this explicitly called out in http://aspnetwebstack.codeplex.com/wikipage?title=CodingConventions.